Urban Mental Health Model (#226)
natcap/invest.users-guide: source/en/urban_mental_health.rst

Residential nature exposure is defined here as the average surrounding greenness near a residence that may contribute to mental health benefits. This exposure is commonly measured using either average normalized difference vegetation index (NDVI, see definition `here <https://www.earthdata.nasa.gov/topics/land-surface/normalized-difference-vegetation-index-ndvi>`__) or the percentage of green space within a specified distance. In this model, average NDVI is used as the primary exposure proxy. This choice is important because health impact assessment should use an exposure metric that is as consistent as possible with the metric used to derive the underlying exposure-response relationship (Giannico et al. 2024). Most of the studies included in the meta-analyses (Rojas-Rueda et al. 2019; Liu et al. 2023) used NDVI as a proxy for nature exposure, typically based on annual average NDVI values.

> To maintain consistency with those effect sizes, we recommend that users provide annual NDVI inputs whenever possible. However, if users have access to effect sizes derived from seasonal NDVI, they may instead use season-specific NDVI inputs. Users should keep in mind that many mental health outcomes reflect longer-term exposures and may not respond strongly to short-term or highly variable changes in greenness. For this reason, the model is best suited for comparing conditions across years, even when season-specific NDVI inputs are used.

Author: @Yingjie4Science Could you please clarify what you mean in the first sentence with "To maintain consistency with those effect sizes"? And/pr provide a clearer transition between the first and second paragraph that better defines effect size?

Reviewer: Basically, the effect size provided by the meta-analysis is about the exposure-response relationship between annual NDVI and health outcomes. So if we use this effect size, we should also use annual NDVI inputs (i.e., To maintain consistency). I'd suggest merge these two paragraph into one to avoid confusion.

Author: Gotcha, so maintaining consistency refers to the _annual_ timescale of NDVI. Since this is the first time the concept of effect size comes up, would it make sense to edit this to (**changes in bold**):

"Most of the studies included in the meta-analyses (Rojas-Rueda et al. 2019; Liu et al. 2023) used NDVI as a proxy for nature exposure, typically based on annual average NDVI values. To maintain consistency with those **annual-scale** effect sizes **(defined here as the magnitude of impact that exposure to NDVI has on a mental health outcome)**, we recommend that users provide annual NDVI inputs whenever possible."

.. _urban_mental_health:

*******************
Urban Mental Health
*******************

Summary
=======

> Nature in urban areas provides important opportunities for recreation and delivers a wide range of physical, social, and psychological health benefits (Bratman et al. 2019; Li et al. 2025). The Urban Mental Health Model was developed to address a critical gap in assessing how changes in urban greening may influence population-level mental health outcomes and reduce the societal costs associated with mental disorders. Alongside other Urban InVEST models, such as Urban Cooling, Urban Flood Risk, Urban Nature Access, and Urban Stormwater, this model expands the Urban InVEST suite by providing spatially explicit estimates of mental health-related benefits, thereby supporting more targeted, equitable, and cost-effective urban greening strategies.

Reviewer 2: Not sure how crucial this is (short forms might be fine here), but I think the official model names are "Urban Flood Risk Mitigation" and "Urban Stormwater Retention" (that's what their UG chapters are called).

```suggestion
Nature in urban areas provides important opportunities for recreation and delivers a wide range of physical, social, and psychological health benefits (Bratman et al. 2019; Li et al. 2025). The Urban Mental Health Model was developed to address a critical gap in assessing how changes in urban greening may influence population-level mental health outcomes and reduce the societal costs associated with mental disorders. Alongside other Urban InVEST models, such as Urban Cooling, Urban Flood Risk Mitigation, Urban Nature Access, and Urban Stormwater Retention, this model expands the Urban InVEST suite by providing spatially explicit estimates of mental health-related benefits, thereby supporting more targeted, equitable, and cost-effective urban greening strategies.
```

The Model
=========

The Urban Mental Health Model estimates the number of preventable mental disorder cases at the pixel level under alternative urban greening scenarios. Based on current data availability and model feasibility, the model focuses on residential nature exposure and uses land use scenarios to characterize changes in greenness exposure and their potential effects on mental health outcomes.

> Residential nature exposure is defined here as the average surrounding greenness near a residence that may contribute to mental health benefits. This exposure is commonly measured using either average normalized difference vegetation index (NDVI, see definition `here <https://www.earthdata.nasa.gov/topics/land-surface/normalized-difference-vegetation-index-ndvi>`__) or the percentage of green space within a specified distance. In this model, average NDVI is used as the primary exposure proxy. This choice is important because health impact assessment should use an exposure metric that is as consistent as possible with the metric used to derive the underlying exposure-response relationship (Giannico et al. 2024). Most of the studies included in the meta-analyses (Rojas-Rueda et al. 2019; Liu et al. 2023) used NDVI as a proxy for nature exposure, typically based on annual average NDVI values.

Reviewer 2: It's always best to make the visible link text descriptive enough to stand on its own without depending on the surrounding context (i.e., "here", while extremely common in the wild, typically isn't sufficient—this is crucial for screen reader users but also helpful for other readers).

The simplest option might be "NDVI definition":

```suggestion
Residential nature exposure is defined here as the average surrounding greenness near a residence that may contribute to mental health benefits. This exposure is commonly measured using either average normalized difference vegetation index (NDVI, see `NDVI definition <https://www.earthdata.nasa.gov/topics/land-surface/normalized-difference-vegetation-index-ndvi>`__) or the percentage of green space within a specified distance. In this model, average NDVI is used as the primary exposure proxy. This choice is important because health impact assessment should use an exposure metric that is as consistent as possible with the metric used to derive the underlying exposure-response relationship (Giannico et al. 2024). Most of the studies included in the meta-analyses (Rojas-Rueda et al. 2019; Liu et al. 2023) used NDVI as a proxy for nature exposure, typically based on annual average NDVI values.
```

Reviewer 2: An even better option would involve being more explicit about the source, for example:

```suggestion
Residential nature exposure is defined here as the average surrounding greenness near a residence that may contribute to mental health benefits. This exposure is commonly measured using either average normalized difference vegetation index (NDVI, for details see `NASA EarthData's Overview of NDVI <https://www.earthdata.nasa.gov/topics/land-surface/normalized-difference-vegetation-index-ndvi>`__) or the percentage of green space within a specified distance. In this model, average NDVI is used as the primary exposure proxy. This choice is important because health impact assessment should use an exposure metric that is as consistent as possible with the metric used to derive the underlying exposure-response relationship (Giannico et al. 2024). Most of the studies included in the meta-analyses (Rojas-Rueda et al. 2019; Liu et al. 2023) used NDVI as a proxy for nature exposure, typically based on annual average NDVI values.
```

The benefit of more detail is that a reader has a clearer sense of where the link goes without having to interact with it.

To maintain consistency with those effect sizes, we recommend that users provide annual NDVI inputs whenever possible. However, if users have access to effect sizes derived from seasonal NDVI, they may instead use season-specific NDVI inputs. Users should keep in mind that some mental health outcomes, especially mental disorders, reflect longer-term exposures and may not respond strongly to short-term or highly variable changes in greenness. For this reason, the model is best suited for comparing conditions across years, even when season-specific NDVI inputs are used.

.. figure:: ./urban_mental_health/residential_nature_exposure.png
   :align: center

> Figure 1. Residential nature exposure is estimated as the average NDVI within a defined buffer distance (or search radius) from each residence (illustration on the left was adapted from Labib et al. 2021).

Reviewer 2: ```suggestion
Figure 1. Residential nature exposure is estimated as the average NDVI within a defined buffer distance (or search radius) from each residence (illustration adapted from Labib et al. 2021).
```

Model Options
-------------

> Land use scenarios are key to understanding how alternative land use change and associated greenery change might impact mental health benefits. To ensure best use of existing science knowledge and better communication with policy makers, we included two options for land use scenario (Figure 2).

Reviewer 2: Should "scenario" at the end of the paragraph be plural? It sounds better plural but I'm not entirely sure it's accurate.

```suggestion
Land use scenarios are key to understanding how alternative land use change and associated greenery change might impact mental health benefits. To ensure best use of existing science knowledge and better communication with policy makers, we included two options for land use scenarios (Figure 2).
```

LULC
~~~~

> The first option for land use scenarios is based on land use and land cover (LULC) maps, which are commonly used by other InVEST models as well as in real-world urban planning practices. Users need to provide the baseline LULC and a LULC under a future or counterfactual scenario in order to get an estimate of the nature exposure difference, which will be measured by NDVI values, based on statistical mapping between land cover types and NDVI values (e.g., forest cover usually has a higher NDVI value than bare land and built-up land). Users therefore need to provide an LULC attribute table, which specifies the average or median NDVI values for each LULC type.

Reviewer 2: Hard to spot in the diff below: I changed an "a" to "an" before "LULC".

```suggestion
The first option for land use scenarios is based on land use and land cover (LULC) maps, which are commonly used by other InVEST models as well as in real-world urban planning practices. Users need to provide the baseline LULC and an LULC under a future or counterfactual scenario in order to get an estimate of the nature exposure difference, which will be measured by NDVI values, based on statistical mapping between land cover types and NDVI values (e.g., forest cover usually has a higher NDVI value than bare land and built-up land). Users therefore need to provide an LULC attribute table, which specifies the average or median NDVI values for each LULC type.
```

NDVI
~~~~

> The second option is similar to the LULC option, with the key difference being that users provide NDVI rasters for both the baseline and scenario conditions. The model directly uses these NDVI rasters to compute neighborhood exposure without requiring LULC-to-NDVI translation.

Reviewer 2: Worth adding future/counterfactual for parallelism with the description of the LULC option?

```suggestion
The second option is similar to the LULC option, with the key difference being that users provide NDVI rasters for both the baseline and future/counterfactual scenario conditions. The model directly uses these NDVI rasters to compute neighborhood exposure without requiring LULC-to-NDVI translation.
```

The following section elaborates the model structure and necessary data inputs as well as the model outputs for interpretation.

.. figure:: ./urban_mental_health/model_options_flowchart.png
   :align: center

> Figure 2. Flowchart illustrates the main computational steps of the Urban Mental Health model, from input data processing to estimation of preventable (or additional) cases and the corresponding changes in health costs. Note: NDVI is used as a proxy of nature exposure. The dashed box indicates optional input data.

Reviewer 2: I imagine others have weighed in on this already, so ignore me if this has already been settled, but… I wonder if it might make more sense to move this figure (and its caption) earlier in this section, just after it's first mentioned and before the "LULC" subsection? I guess it depends on whether you want to present the text description first or the visual first; and/or it might come down to personal preference. I just found it a little surprising to see a reference to Figure 2 followed by a few paragraphs instead of being immediately followed by Figure 2 itself.

Author: This was not discussed, so I will go ahead and move it up before the LULC heading. Thanks!

The baseline nature exposure is the actual level of greenness as measured by the average NDVI within a user-defined search radius (or buffer distance) around each residence (here, we use population pixel as the proxy in this model). Given the potential mental health benefits of blue spaces, it would be ideal to include them in exposure assessments to more fully capture the diverse contributions of natural environments to human well-being. However, water bodies typically exhibit negative NDVI values, and there is currently insufficient empirical scientific evidence to derive a reliable effect size for blue space exposure using NDVI as a proxy. Consequently, the current version of the model provides an option to exclude (mask out) pixels classified as water bodies from the greenness exposure assessment. However, if users have access to robust, peer-reviewed effect sizes linking NDVI-based blue space exposure to health outcomes, they may choose to retain water bodies for a seperate model run and analysis.

> Nature exposure under a scenario condition can be estimated using one of the two options elaborated as follows, designed to leverage the best available scientific knowledge while enhancing communication with policymakers. Each option is detailed in the following subsections.

Reviewer 2: Add future/counterfactual here as well? Or does this refer to any scenario?

```suggestion
Nature exposure under a future/counterfactual scenario condition can be estimated using one of the two options elaborated as follows, designed to leverage the best available scientific knowledge while enhancing communication with policymakers. Each option is detailed in the following subsections.
```

Author: Good question, I took this to refer to either the baseline scenario or future/counterfactual scenario since we do ultimately produce nature exposure maps for both scenarios

NE estimation [option 1] - Translating LULC to NDVI with attribute mapping
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> The first approach estimates scenario-based nature exposure using land use and land cover (LULC) maps, which are widely used in other InVEST models and real-world urban planning applications. To apply this method, users must provide both a baseline LULC map and a scenario LULC map (e.g., reflecting future or counterfactual conditions). The change in nature exposure (ΔNE) is then derived by translating LULC categories into corresponding NDVI values, based on statistical relationships between land cover types and NDVI. This relationship can either be derived by the user before running the model, or can be automatically calculated by the model if the user provide an NDVI map. For example, forested areas typically exhibit higher NDVI values than bare or built-up land.

Reviewer 2: "provide" -> "provides"

```suggestion
The first approach estimates scenario-based nature exposure using land use and land cover (LULC) maps, which are widely used in other InVEST models and real-world urban planning applications. To apply this method, users must provide both a baseline LULC map and a scenario LULC map (e.g., reflecting future or counterfactual conditions). The change in nature exposure (ΔNE) is then derived by translating LULC categories into corresponding NDVI values, based on statistical relationships between land cover types and NDVI. This relationship can either be derived by the user before running the model, or can be automatically calculated by the model if the user provides an NDVI map. For example, forested areas typically exhibit higher NDVI values than bare or built-up land.
```

> To enable this translation, users are required to supply a LULC attribute table that assigns an average or median NDVI value to each LULC type (see Table 1 for an example). Using this attribute table, the model computes :math:`NE_{baseline}` and :math:`NE_{scenario}` from the respective LULC maps, allowing for an estimation of ΔNE that reflects the projected landscape changes.

Reviewer 2: "a" -> "an"

```suggestion
To enable this translation, users are required to supply an LULC attribute table that assigns an average or median NDVI value to each LULC type (see Table 1 for an example). Using this attribute table, the model computes :math:`NE_{baseline}` and :math:`NE_{scenario}` from the respective LULC maps, allowing for an estimation of ΔNE that reflects the projected landscape changes.
```

The LULC-based scenarios usually include not only the greening conversion but also the conversion of natural or vacant land into built-up areas. In such cases, the model allows for negative values in ΔNE, capturing potential declines in nature exposure. This enables policymakers to assess not only the potential health benefits of urban greening but also the health risks associated with land use changes that reduce access to greenspace.

Table 1. Sample Land use and land cover (LULC) attribute table.

====== ============================ ======= =======
lucode lu_desc                      exclude ndvi
====== ============================ ======= =======
11     Open water                   1       -0.1437
21     Developed, open space        0       0.5873
22     Developed, low intensity     0       0.3773
23     Developed, medium intensity  0       0.2305

> 24     Developed high intensity     0       0.1040

Reviewer 2: ```suggestion
24     Developed, high intensity     0       0.1040
```

31     Barren land (rock/sand/clay) 0       0.1700
41     Deciduous forest             0       0.7493
42     Evergreen forest             0       0.6956
43     Mixed forest                 0       0.6931
52     Shrub/scrub                  0       0.6074
71     Grassland/herbaceous         0       0.5289
81     Pasture/hay                  0       0.5167
82     Cultivated crops             0       0.4255
90     Woody wetlands               1       0.5779
95     Emergent herbaceous wetlands 1       0.2792
====== ============================ ======= =======

> Note: The \`exclude\` field identifies LULC classes that should be excluded (masked out, indicated by a value of 1) from the greenness exposure assessment. We recommend excluding water bodies, as they typically exhibit negative NDVI values and there is currently insufficient empirical evidence to establish a reliable effect size for blue space exposure using NDVI as a proxy. \`ndvi\` field specifies the average or median NDVI value associated with each LULC class. Users can derive these values by overlaying a LULC raster and an NDVI raster from the same time period (e.g., the same year or growing season) to ensure temporal consistency.

Reviewer 2: ```suggestion
Note: The \`exclude\` field identifies LULC classes that should be excluded (masked out, indicated by a value of 1) from the greenness exposure assessment. We recommend excluding water bodies, as they typically exhibit negative NDVI values and there is currently insufficient empirical evidence to establish a reliable effect size for blue space exposure using NDVI as a proxy. The \`ndvi\` field specifies the average or median NDVI value associated with each LULC class. Users can derive these values by overlaying an LULC raster and an NDVI raster from the same time period (e.g., the same year or growing season) to ensure temporal consistency.
```

NE estimation [option 2] - Using NDVI at two time points directly
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> The second approach is similar to the LULC option, but with a key difference: users directly provide NDVI rasters for both baseline and scenario conditions. This option is particularly useful when users wish to use historical or observed data to assess the potential health benefits or risks associated with past or projected landscape changes.

Reviewer 2: Add future/counterfactual here?

```suggestion
The second approach is similar to the LULC option, but with a key difference: users directly provide NDVI rasters for both baseline and future/counterfactual scenario conditions. This option is particularly useful when users wish to use historical or observed data to assess the potential health benefits or risks associated with past or projected landscape changes.
```

NDVI-based :math:`NE_{baseline}` and :math:`NE_{scenario}` are calculated as the average NDVI within a user-defined search radius around each population pixel. The model then computes the change in nature exposure (:math:`\Delta NE`) to be used to estimate the health impacts of landscape transformation.

> Water and other land cover classes can be masked out in the same manner as Option 1, i.e., based on a LULC attribute table and baseline (and optionally, alternate) LULC map provided by users. If LULC inputs are not provided, NDVI pixels that are less than 0 will be masked out to exclude water bodies from the analysis (given that NDVI values for water typically fall below 0).

Reviewer 2: ```suggestion
Water and other land cover classes can be masked out in the same manner as in Option 1, i.e., based on a user-provided LULC attribute table, baseline LULC map, and (optional) alternate LULC map. If LULC inputs are not provided, NDVI pixels that are less than 0 will be masked out to exclude water bodies from the analysis (given that NDVI values for water typically fall below 0).
```

Health Impact Assessment
------------------------

The model conducts a quantitative health impact assessment at the pixel level to estimate the number of preventable mental health cases resulting from changes in nature exposure. Specifically, it compares exposure under a baseline scenario with that of a counterfactual or target greening scenario.

When estimating preventable cases (:math:`PC`), the model first calculates baseline cases (:math:`BC`) for each spatial unit (e.g., region, district, or census tract), using either observed case counts or prevalence rates. If only baseline prevalence rates are available, users must provide spatialized population raster data to derive baseline case estimates.

> For each pixel, the model computes the change in nature exposure (:math:`\Delta NE`) by subtracting baseline exposure (:math:`NE_{baseline}`) from the counterfactual or scenario value (:math:`NE_{scenario}`). The model then applies a dose–response function, expressed as a relative risk (:math:`RR`) per unit change in exposure (e.g., :math:`RR_{0.1NE}`), to quantify the health impact.

Reviewer 2: Add "target greening" for parallelism with first paragraph in this subsection?
Also looks like `dose–response` contains an en-dash; is that needed here or is a hyphen sufficient?

```suggestion
For each pixel, the model computes the change in nature exposure (:math:`\Delta NE`) by subtracting baseline exposure (:math:`NE_{baseline}`) from the counterfactual or target greening scenario value (:math:`NE_{scenario}`). The model then applies a dose-response function, expressed as a relative risk (:math:`RR`) per unit change in exposure (e.g., :math:`RR_{0.1NE}`), to quantify the health impact.
```

where

- :math:`PC`: preventable cases
- :math:`PF`: preventable fraction
- :math:`BC`: baseline cases. By default, baseline cases are calculated using the baseline prevalence rate and population data provided by the user.
- :math:`BIR`: baseline prevalence rate. Input data provided by users, typically sourced from national or local health agencies such as the U.S. Centers for Disease Control and Prevention (CDC). These rates are often derived from survey-based statistics and are available at various spatial levels (e.g., region, district, or census tract). If the data is provided in tabular format rather than as shapefiles, users must join it to the corresponding spatial units before using it as model input.
- :math:`POP`: population. A raster dataset provided by the user, representing the number of inhabitants per pixel across the study area. Pixels with no population should be assigned a value of 0.
- :math:`RR`: relative risk or risk ratio. A value less than 1 (:math:`RR` < 1) indicates that nature exposure is associated with a reduced risk of mental illness (protective effect), whereas a value greater than 1 (:math:`RR` > 1) indicates an increased risk (adverse association).

> - :math:`NE`: nature exposure. Here, the model uses NDVI as a proxy. :math:`NE_{baseline}` is provided by the user as input, representing baseline nature exposure levels. :math:`NE_{scenario}` is typically calculated within the model based on the land use/land cover (LULC) scenario map provided by users, allowing for assessment of changes in exposure under alternative greening strategies (see Nature Exposure Estimation section for details).

Reviewer 2: Not a crucial change; "user-provided" just sounds a little more natural to my ear than "provided by users".

```suggestion
- :math:`NE`: nature exposure. Here, the model uses NDVI as a proxy. :math:`NE_{baseline}` is provided by the user as input, representing baseline nature exposure levels. :math:`NE_{scenario}` is typically calculated within the model based on the user-provided land use/land cover (LULC) scenario map, allowing for assessment of changes in exposure under alternative greening strategies (see Nature Exposure Estimation section for details).
```

- :math:`RR_{0.1NE}`: Relative risk per 0.1 :math:`NE` increase. By default, the model uses the relative risk associated with a 0.1 increase in NDVI-based nature exposure. This value must be provided by users, based on empirical studies or meta-analyses relevant to the selected health outcome. In the sample data, we derived effect sizes for depression and anxiety from a meta-analysis by Liu et al. 2023.

> If the risk ratio is not available, while other effect size metrics, such as the odds ratio (:math:`OR`) is available, a commonly used method to approximate the :math:`RR` from an :math:`OR` is based on a formula proposed by Zhang and Yu (1998). This formula requires you to know or estimate the baseline risk (the prevalence of the outcome in the reference or non-exposed group).

Reviewer 2: ```suggestion
If the risk ratio is not available, but other effect size metrics, such as the odds ratio (:math:`OR`) is available, a commonly used method to approximate the :math:`RR` from an :math:`OR` is based on a formula proposed by Zhang and Yu (1998). This formula requires you to know or estimate the baseline risk (the prevalence of the outcome in the reference or non-exposed group).
```

.. math:: RR = \frac{OR}{1 - p_0 + (p_0 \cdot OR)}
	:label: (umh. 5)

Where :math:`p_0` is the baseline risk. In this context, this should be the probability of mental disorder for a population without nature exposure or with very limited nature exposure.

Health Cost Estimation
----------------------

To quantify the economic value of improved mental health outcomes, the model estimates the avoided health costs associated with the number of preventable cases identified in the health impact assessment.

> For each spatial unit or pixel, the model multiplies the estimated number of preventable cases (:math:`PC`) by the corresponding cost per case (e.g., in USD Purchasing Power Parity (PPP) or local currency units per user-defined), as shown in the equation below. This cost can be based on any cost estimates that are available to the user. For societal cost used in the sample data, it represents the average economic burden of a single case of a given mental health outcome (e.g., depression or anxiety), including direct healthcare costs, productivity losses, and other social costs. If users cannot access societal cost, they can specify any format of estimate for this calculation.

Reviewer 2: Removed "per user-defined" because it wasn't quite right grammatically and I couldn't come up with a way to reword it that wasn't overly clunky. Given the additional context, I think it's OK without.

```suggestion
For each spatial unit or pixel, the model multiplies the estimated number of preventable cases (:math:`PC`) by the corresponding cost per case (e.g., in USD Purchasing Power Parity (PPP) or local currency units), as shown in the equation below. This cost can be based on any cost estimates that are available to the user. The societal cost used in the sample data represents the average economic burden of a single case of a given mental health outcome (e.g., depression or anxiety), including direct healthcare costs, productivity losses, and other social costs. If users cannot access societal cost, they can specify any format of estimate for this calculation.
```

.. math:: Avoided Cost = Preventable Cases \times Health Cost Rate
	:label: (umh. 6)

.. note:: A negative value for "Avoided Cost" indicates an "Additional Cost" compared to the baseline condition.

> Users are required to provide a health cost rate value, which indicates illness-specific and, where available, country- or region-specific estimates of cost per case. A global meta-analysis (Christensen et al. 2020) reports societal costs of mental disorders across more than 30 countries. For instance, the societal cost per patient in the USA was estimated to be 11,000 USD PPP in the year of 2018. Where more detailed or local data are available, users are encouraged to collect such data values for increased accuracy. In the absence of granular data, national or regional averages may serve as reasonable substitutes. Given the model's focus on urban studies and the limited availability of health cost data at the sub-urban level, the current version supports uniform cost inputs at the city level. However, users may supply varying cost data across different cities to enable comparisons across diverse geographic and economic contexts. This step allows users to assess not only the health implications of urban greening scenarios, but also their potential economic co-benefits, supporting cost–benefit analyses for policy and planning decisions.

Reviewer 2: Removed "of" from "year of 2018" and "values" from "collect such data values":

```suggestion
Users are required to provide a health cost rate value, which indicates illness-specific and, where available, country- or region-specific estimates of cost per case. A global meta-analysis (Christensen et al. 2020) reports societal costs of mental disorders across more than 30 countries. For instance, the societal cost per patient in the USA was estimated to be 11,000 USD PPP in the year 2018. Where more detailed or local data are available, users are encouraged to collect such data for increased accuracy. In the absence of granular data, national or regional averages may serve as reasonable substitutes. Given the model's focus on urban studies and the limited availability of health cost data at the sub-urban level, the current version supports uniform cost inputs at the city level. However, users may supply varying cost data across different cities to enable comparisons across diverse geographic and economic contexts. This step allows users to assess not only the health implications of urban greening scenarios, but also their potential economic co-benefits, supporting cost–benefit analyses for policy and planning decisions.
```

Limitations and Simplifications
===============================

> Nature exposure can be characterized in several ways, including availability, accessibility or proximity (for example, as estimated by the `Urban Nature Access <https://naturalcapitalalliance.stanford.edu/invest/urban-nature-access>`__ model), and intensity of nature contact. This model focuses primarily on residential greenness, which reflects the availability dimension of nature exposure and is one of the most commonly studied indicators in the literature on urban nature and mental health. As a result, the current model does not account for: (1) blue space exposures, such as water bodies and their potential effects on mental health; and (2) other dimensions of nature exposure, including accessibility and intensity of contact, because the available evidence linking these metrics to mental health outcomes remains more limited.

Reviewer 2: I think there's an extra `_` at the end of the UNA link markup.

```suggestion
Nature exposure can be characterized in several ways, including availability, accessibility or proximity (for example, as estimated by the `Urban Nature Access <https://naturalcapitalalliance.stanford.edu/invest/urban-nature-access>`_ model), and intensity of nature contact. This model focuses primarily on residential greenness, which reflects the availability dimension of nature exposure and is one of the most commonly studied indicators in the literature on urban nature and mental health. As a result, the current model does not account for: (1) blue space exposures, such as water bodies and their potential effects on mental health; and (2) other dimensions of nature exposure, including accessibility and intensity of contact, because the available evidence linking these metrics to mental health outcomes remains more limited.
```

Data Needs
==========

- :investspec:`urban_mental_health workspace_dir`

- :investspec:`urban_mental_health results_suffix`

- :investspec:`urban_mental_health aoi_path`

- :investspec:`urban_mental_health population_raster`

- :investspec:`urban_mental_health search_radius`

- :investspec:`urban_mental_health effect_size`

- :investspec:`urban_mental_health baseline_prevalence_vector`

- :investspec:`urban_mental_health health_cost_rate`

- :investspec:`urban_mental_health model_option`

- :investspec:`urban_mental_health ndvi_base`

- :investspec:`urban_mental_health ndvi_alt`

- :investspec:`urban_mental_health lulc_base`

- :investspec:`urban_mental_health lulc_alt`

- :investspec:`urban_mental_health lulc_attr_csv`

  Columns:

  - :investspec:`urban_mental_health lulc_attr_csv.columns.lucode`

  - :investspec:`urban_mental_health lulc_attr_csv.columns.exclude`

  - :investspec:`urban_mental_health lulc_attr_csv.columns.ndvi`

Model Outputs
=============

> .. note:: If the model option 'NDVI' is used, outputs will be aligned to the Baseline NDVI raster grid and the target pixel size will be derived from that raster. If LULC is used as the model option, the Baseline LULC raster will serve as the alignment raster and will define the target pixel size. The AOI provides the target projection and extent.

Reviewer 2: Minor edits for parallelism and clarity:

```suggestion
.. note:: If the 'NDVI' model option is used, outputs will be aligned to the Baseline NDVI raster grid, and the target pixel size will be derived from that raster. If the 'LULC' model option is used, the Baseline LULC raster will serve as the alignment raster and will define the target pixel size. In both cases, the AOI provides the target projection and extent.
```

* **urban_mental_health_report.html:** A summary of a model run, including visualizations of key outputs, tables of calculated results, and information about model inputs. The report can be accessed in the Workbench or opened with any web browser. For an example, see the `Sample Urban Mental Health Report <https://storage.googleapis.com/data.naturalcapitalproject.org/invest-reports/urban_mental_health_report.html>`_, generated by running the Urban Mental Health model with the Urban Mental Health sample data.

* **preventable_cases.tif** (Raster, count): Pixel-level estimates of preventable cases.

* **preventable_cost.tif** (Raster, currency unit): Pixel-level estimates of preventable costs. The sample data use societal cost per case, expressed in 2018 USD PPP. Users may customize the currency unit and reference year in the health cost rate table to better align with local policy priorities and economic conditions.

> * **preventable_cases_cost_sum.gpkg** (Shapefile): Aggregated total preventable cases and total preventable costs by subregion (for example, census tract or ZIP code) within the area of interest. The area of interest input is used for this aggregation. The units for cases and cost are the same as \`preventable_cases.tif\` and \`preventable_cost.tif\`.

Reviewer 2: ```suggestion
* **preventable_cases_cost_sum.gpkg** (GeoPackage): Aggregated total preventable cases and total preventable costs by subregion (for example, census tract or ZIP code) within the area of interest. The area of interest input is used for this aggregation. The units for cases and cost are the same as \`preventable_cases.tif\` and \`preventable_cost.tif\`.
```

> * **preventable_cases_cost_sum.gpkg** (Shapefile): Aggregated total preventable cases and total preventable costs by subregion (for example, census tract or ZIP code) within the area of interest. The area of interest input is used for this aggregation. The units for cases and cost are the same as \`preventable_cases.tif\` and \`preventable_cost.tif\`.

Reviewer 2: Or do we typically keep this designation generic and just say "Vector" regardless of the specific file type?

```suggestion
* **preventable_cases_cost_sum.gpkg** (Vector): Aggregated total preventable cases and total preventable costs by subregion (for example, census tract or ZIP code) within the area of interest. The area of interest input is used for this aggregation. The units for cases and cost are the same as \`preventable_cases.tif\` and \`preventable_cost.tif\`.
```

Author: Looks like we usually don't specify and leave it to the user to interpret the type from the extension, but I kinda like being explicit here because we do have 2 similarly-named outputs (a CSV and a GPKG).

* **ndvi_base_aligned_masked.tif** (Raster): Baseline NDVI raster after excluded LULC classes have been masked out.

* **ndvi_base_buffer_mean.tif** (Raster): Baseline neighborhood greenness surface, calculated as the mean NDVI within the specified search radius. This represents baseline nature exposure.

* **ndvi_alt_buffer_mean.tif** (Raster): Alternative-scenario neighborhood greenness surface, calculated as the mean NDVI within the specified search radius. This represents scenario nature exposure.

* **ndvi_base_buffer_mean_clipped.tif** (Raster): Baseline neighborhood greenness surface, i.e., nature exposure, clipped to the analysis extent.

* **ndvi_alt_buffer_mean_clipped.tif** (Raster): Alternative-scenario neighborhood greenness surface, i.e., nature exposure, clipped to the analysis extent.

* **delta_ndvi.tif** (Raster): Pixel-level change in NDVI, as a proxy for nature exposure, between the alternative and baseline scenarios. This is calculated as ndvi_alt_buffer_mean_clipped.tif minus ndvi_base_buffer_mean_clipped.tif.

* **population_aligned.tif** (Raster, count): Population raster aligned to the model's working grid.

> * **population_aligned_clipped.tif** (Raster, count): Population raster clipped to the analysis extent

Reviewer 2: ```suggestion
* **population_aligned_clipped.tif** (Raster, count): Population raster clipped to the analysis extent.
```

Appendix: Data Sources
======================

:ref:`Land Use/Land Cover <lulc>`
---------------------------------

NDVI
----

NDVI at a resolution of 30 m can be derived from the Landsat image collections via Google Earth Engine. Users can also use Sentinel-2 or MODIS (or Moderate Resolution Imaging Spectroradiometer) imagery to derive NDVI rasters.

> Population raster

Reviewer 2: Most of the other subheadings here seem to be using title case, so I'd update this to match.

```suggestion
Population Raster
```

-----------------

Multiple regional and global datasets exist that estimate population size and density at high resolution, such as:

- WorldPop global population data: https://www.worldpop.org/methods/populations/. For population count rasters, users are recommended to use the 100 meter resolution one: https://hub.worldpop.org/project/categories?id=3

> - Meta/CIESIN global population density data: https://dataforgood.facebook.com/dfg/tools/high-resolution-population-density-maps

Reviewer 2: I recommend using your descriptive text as the visible link text rather than raw URLs, e.g.:

```suggestion
- `WorldPop global population data <https://www.worldpop.org/methods/populations/>`_. For population count rasters, users are recommended to use a `WorldPop population raster with 100-meter resolution <https://hub.worldpop.org/project/categories?id=3>`_.
- `Meta/CIESIN global population density data <https://dataforgood.facebook.com/dfg/tools/high-resolution-population-density-maps>`_
```

Also, it looks like the page that specifically lists 100m country-specific WorldPop rasters is `https://hub.worldpop.org/geodata/listing?id=135`. You may want to link there directly, unless the category page (currently linked) provides important additional context.

Search Radius
-------------

> A common approach to assessing nature or greenspace exposure is to estimate vegetation availability (e.g., greenness) within buffers (search radius) around locations where people live or spend time. Users should define this radius based on human mobility patterns in their study area or draw on parameters used in empirical studies. For example, Zare Sakhvidi et al. (2025) provides methodological guidance for selecting appropriate buffer sizes (see https://doi.org/10.1016/j.lanplh.2025.101370).

Reviewer 2: ```suggestion
A common approach to assessing nature or greenspace exposure is to estimate vegetation availability (e.g., greenness) within buffers (search radius) around locations where people live or spend time. Users should define this radius based on human mobility patterns in their study area or draw on parameters used in empirical studies. For example, Zare Sakhvidi et al. (2025) provide methodological guidance for selecting appropriate buffer sizes (see `"Methodological guidance for selecting buffers in greenspace–health studies" <https://doi.org/10.1016/j.lanplh.2025.101370>`_).
```

Health Effect Size
------------------

The health effect size is indicator-specific and represents the relationship between nature exposure and the mental health outcome. It is given as relative risk associated with a 0.1 increase in NDVI. In the sample data, we derived effect size for depression from a meta-analysis by Liu et al. 2023. If the user has an effect size value as an odds ratio (OR), a commonly used method to approximate the relative risk from an OR is based on a formula proposed by Zhang and Yu (1998). This formula requires you to know or estimate the baseline risk (the prevalence of the outcome in the reference or non-exposed group). See Health Impact Assessment section for details.

Baseline Prevalence
-------------------

> Baseline prevalence represents the rate of a specific mental health outcome (e.g., depression or anxiety) across administrative units within the study area. This data allows the model to estimate preventable cases by comparing current rates with those projected under improved nature exposure scenarios

Reviewer 2: ```suggestion
Baseline prevalence represents the rate of a specific mental health outcome (e.g., depression or anxiety) across administrative units within the study area. This data allows the model to estimate preventable cases by comparing current rates with those projected under improved nature exposure scenarios.
```

> Health cost rate

Reviewer 2: ```suggestion
Health Cost Rate
```

----------------

> The health cost rate represents the illness-specific and, where available, country- or region-specific estimates of cost per case. A global meta-analysis (Christensen et al. 2020) reports societal costs of mental disorders across more than 30 countries. For instance, the societal cost per patient in the USA was estimated to be 11,000 USD PPP in the year of 2018.

Reviewer 2: ```suggestion
The health cost rate represents the illness-specific and, where available, country- or region-specific estimates of cost per case. A global meta-analysis (Christensen et al. 2020) reports societal costs of mental disorders across more than 30 countries. For instance, the societal cost per patient in the USA was estimated to be 11,000 USD PPP in the year 2018.
```

References
==========

Bratman GN, Anderson CB, Berman MG, et al (2019) Nature and mental health: An ecosystem service perspective. Science Advances 5:eaax0903. https://doi.org/10.1126/sciadv.aax0903

> Christensen MK, Lim CCW, Saha S, et al (2020) The cost of mental disorders: a systematic review. Epidemiology and Psychiatric Sciences 29:e161. https://doi.org/10.1017/S204579602000075X

Reviewer 2: Some of these sources that include `:e` are triggering some RST syntax highlighting. It might be worth double-checking this section in the rendered HTML just to make sure everything looks right!

Author: Everything looks fine to me!

Giannico OV, Sardone R, Bisceglia L, et al (2024) The mortality impacts of greening Italy. Nat Commun 15:10452. https://doi.org/10.1038/s41467-024-54388-7

Labib SM, Huck JJ, Lindley S (2021) Modelling and mapping eye-level greenness visibility exposure using multi-source data at high spatial resolutions. Science of The Total Environment 755:143050. https://doi.org/10.1016/j.scitotenv.2020.143050

Li Y, Mao Y, Mandle L, et al (2025) Acute mental health benefits of urban nature. Nat Cities 2:720-731. https://doi.org/10.1038/s44284-025-00286-y

Liu Z, Chen X, Cui H, et al (2023) Green space exposure on depression and anxiety outcomes: A meta-analysis. Environmental Research 231:116303. https://doi.org/10.1016/j.envres.2023.116303

Rojas-Rueda D, Nieuwenhuijsen MJ, Gascon M, et al (2019) Green spaces and mortality: a systematic review and meta-analysis of cohort studies. The Lancet Planetary Health 3:e469-e477. https://doi.org/10.1016/S2542-5196(19)30215-3

> 

Reviewer 2: Should we also include the Zare Sakhvidi et al publication mentioned in the "Search Radius" subsection of the Appendix? I'm not sure if this list is typically limited to sources referenced in the model-specific discussion or not.

Author: I has a similar thought, but maybe I'll play it safe and add it? I don't really see the harm

> Zhang J, Yu KF (1998) What's the Relative Risk?A Method of Correcting the Odds Ratio in Cohort Studies of Common Outcomes. JAMA 280:1690-1691. https://doi.org/10.1001/jama.280.19.1690

Reviewer 2: ```suggestion
Zhang J, Yu KF (1998) What's the Relative Risk? A Method of Correcting the Odds Ratio in Cohort Studies of Common Outcomes. JAMA 280:1690-1691. https://doi.org/10.1001/jama.280.19.1690
```

Christensen MK, Lim CCW, Saha S, et al (2020) The cost of mental disorders: a systematic review. Epidemiology and Psychiatric Sciences 29:e161. https://doi.org/10.1017/S204579602000075X

Giannico OV, Sardone R, Bisceglia L, et al (2024) The mortality impacts of greening Italy. Nat Commun 15:10452. https://doi.org/10.1038/s41467-024-54388-7

Labib SM, Huck JJ, Lindley S (2021) Modelling and mapping eye-level greenness visibility exposure using multi-source data at high spatial resolutions. Science of The Total Environment 755:143050. https://doi.org/10.1016/j.scitotenv.2020.143050

Li Y, Mao Y, Mandle L, et al (2025) Acute mental health benefits of urban nature. Nat Cities 2:720-731. https://doi.org/10.1038/s44284-025-00286-y

Liu Z, Chen X, Cui H, et al (2023) Green space exposure on depression and anxiety outcomes: A meta-analysis. Environmental Research 231:116303. https://doi.org/10.1016/j.envres.2023.116303

Rojas-Rueda D, Nieuwenhuijsen MJ, Gascon M, et al (2019) Green spaces and mortality: a systematic review and meta-analysis of cohort studies. The Lancet Planetary Health 3:e469-e477. https://doi.org/10.1016/S2542-5196(19)30215-3

> Zare Sakhvidi M, Browning, M, Samuelsson, K, et al (2025) Methodological guidance for selecting buffers in greenspace-health studies. The Lancet Planetary Health 9:e101370. https://doi.org/10.1016/j.lanplh.2025.101370

Reviewer 2: It looks like the other sources listed here don't include commas between last names and first initials. Should we update this one to match? (I honestly don't know what's correct in this particular citation style.)

```suggestion
Zare Sakhvidi M, Browning M, Samuelsson K, et al (2025) Methodological guidance for selecting buffers in greenspace-health studies. The Lancet Planetary Health 9:e101370. https://doi.org/10.1016/j.lanplh.2025.101370
```

Author: I'm also not sure what this citation style is, but I could add commas to everything? It doesn't look like we're totally consistent across the different chapters either

Reviewer 2: If there's no clear consensus among existing chapters, then I think either option (removing commas from this entry, or adding them to all the others) would be fine—I'll leave it up to you.

Author: Went with the dominant style in the other UG chapters!

Reviewer 2: Thanks! This change brought some bonus improvements along with it (like ditching those abbreviated journal titles that were bugging me!).
